.vscode/tasks.json

```diff
@@ -42,8 +42,15 @@
     },
     {
       "label": "Debug: Excel Desktop",
-      "type": "npm",
-      "script": "start:desktop -- --app excel",
+      "type": "shell",
+      "command": "npm",
+      "args": [
+        "run",
+        "start:desktop",
+        "--",
+        "--app",
+        "excel",
```

Comment:
Need to remove the trailing comma on the final element of the array.  Technically this is valid for VS Code's use of tasks.json, but it's not valid json and our convert script tries to parse this file as json and fails.

Reply:
Hmmm that wasn't the only spot. I ran the file through a validator so it should be good now. thanks!

Reply:
Yeah . . . I thought would figure out all the places needed so I didn't comment on each one.